3-With-Intune MAM/IntuneClient/active-directory-xamarin-intune.iOS/active-directory-xamarin-intune.iOS.csproj

```diff
@@ -127,13 +129,10 @@
     <Reference Include="System.Numerics.Vectors" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Identity.Client">
-      <Version>4.44.0</Version>
-    </PackageReference>
     <PackageReference Include="Xamarin.Forms" Version="5.0.0.2012" />
     <PackageReference Include="Xamarin.Essentials" Version="1.6.1" />
     <PackageReference Include="Microsoft.Intune.MAM.Xamarin.iOS">
-      <Version>15.3.0.1</Version>
+      <Version>16.0.1.2</Version>

```

Comment:
This version of the Intune Xamarin bindings does not exist in production.

Reply:
Updated to 16.0.3